input/fsh/Alias.fsh

```diff
@@ -7,12 +7,12 @@
 
 Alias: HL7V2 = http://terminology.hl7.org/CodeSystem/v2-0203
 Alias: CitizenshipExtension = http://hl7.org/fhir/StructureDefinition/patient-citizenship
-Alias: ELGALaendercodesVS = http://hl7.at/fhir/HL7ATCoreProfiles/4.0.1/ValueSet/at-elga-vs-countryCodes
-Alias: HL7AustriaReligionVS = http://hl7.at/fhir/HL7ATCoreProfiles/4.0.1/ValueSet/at-elga-vs-religion
+Alias: ELGALaendercodesVS = https://termgit.elga.gv.at/ValueSet-elga-laendercodes.html
+Alias: ELGAAustriaReligionVS = https://termgit.elga.gv.at/ValueSet-elga-religiousaffiliation.html

```

Comment:
Should the canonical of the two value sets be used here? i.e.:

- https://termgit.elga.gv.at/ValueSet/elga-laendercodes
- https://termgit.elga.gv.at/ValueSet/elga-religiousaffiliation